tests/test_aggregates.py

```diff
@@ -10,6 +10,7 @@
     ScheduleEByCandidateView,
     CommunicationCostByCandidateView,
     ElectioneeringByCandidateView,
+    ScheduleAByStateView,
```

Comment:
It would be easier for reading and maintenance if these were sorted.

Reply:
good suggestion.